app/controllers/project/project_grant_request_controller.rb

```diff
@@ -5,8 +5,11 @@ def show
 
     @total_project_cost = helpers.calculate_total(@project.project_costs).to_i
     @total_cash_contributions = helpers.calculate_total(@project.cash_contributions).to_i
-
     @final_grant_amount = @total_project_cost - @total_cash_contributions
+    @maximum_request_value = 10000.to_i
+    @costs_greater_than_contributions = @final_grant_amount.positive? ? true : false
```

Comment:
```suggestion
    @costs_greater_than_contributions = @final_grant_amount.positive? 
```
I think we can remove this as it already returns a boolean